java/server/src/main/java/com/squareup/subzero/server/resources/GenerateQrCodeResource.java

```diff
@@ -144,15 +146,12 @@ public QrCode signTxRequest(SignTxRequest request) throws WriterException {
     builder.setSignTx(signTxBuilder);
 
     byte [] command_bytes = builder.build().toByteArray();
-    //TODO: This will be used later to add another test negative test case.
-    // assign an unexpected protobuf schema to make the decoding fail in subzero core.
-    //Internal.InternalCommandRequest.Builder temp_builder = Internal.InternalCommandRequest.newBuilder();
-    //temp_builder.setInitWallet(Internal.InternalCommandRequest.InitWalletRequest.newBuilder());
-    //command_bytes = temp_builder.build().toByteArray();
-    //Sign with a know dev key. This is for testing only.
+    // Sign with a known dev key. This is for testing only.
```

Comment:
Deleting comments that did not really end up help in generating a negative test case.